### PR TITLE
Fix pdf panel download

### LINKED ISSUE #4930: Panel export in PDF format is missing a number of fields/keys
image

### PR DESCRIPTION
**Pull request by northwestwitch**
This PR adds a functionality or fixes a bug.
- Fix #4930 

### Before:

image

### After:

image

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DN
- [x] tests executed by DN

### CODE REVIEW

File: scout/server/blueprints/panels/templates/panels/panel_pdf_simple.html [hunk 1]
```diff
@@ -18,25 +18,28 @@ <h4>Scout - Gene panel report</h4> - created on:&nbsp;<strong>{{report_created_a
   <div class="card-body">
     <table class="table table-sm">
       <tr>
-        <td>Updated:<span class="badge bg-secondary rounded-pill text-white">{{ panel.date.strftime('%Y-%m-%d') }}</span></td>
```
Comment (northwestwitch) [inline scout/server/blueprints/panels/templates/panels/panel_pdf_simple.html:21]: The fix is the style of these spans, which is no longer recognized. I've devided to remove the spans, so it's easier to maintain

File: scout/server/blueprints/panels/templates/panels/panel_pdf_simple.html [hunk 2]
```diff
@@ -18,25 +18,28 @@ <h4>Scout - Gene panel report</h4> - created on:&nbsp;<strong>{{report_created_a
...
+            False
           {% endif %}
         </td>
       </tr>
       <tr>
-        <td>Panel database ID: <span class="badge bg-secondary rounded-pill text-white">{{ panel._id }}</span></td>
+        <td>Panel database ID: {{ panel._id }}</td>
+      </tr>
+      <tr>
+        <td>Maintainers: {{ panel.maintainer|join(", ") }}</td>
```
Comment (northwestwitch) [inline scout/server/blueprints/panels/templates/panels/panel_pdf_simple.html:42]: Added new

File: scout/server/blueprints/panels/templates/panels/panel_pdf_simple.html [hunk 3]
```diff
@@ -45,27 +48,31 @@ <h4>Scout - Gene panel report</h4> - created on:&nbsp;<strong>{{report_created_a
...
+        <th>Manual inheritance (pre-set terms)</th>
+        <th>Manual inheritance (free text terms)</th>
+        <th>Comment</th>
       </tr>
       {% for gene in panel.genes|sort(attribute='symbol') %}
         <tr>
           <td>{{loop.index}}</td>
+          <td>{{ gene.symbol }}</td>
           <td>
-            <a style="text-decoration:none;" href="https://www.genenames.org/cgi-bin/gene_symbol_report?hgnc_id={{gene.hgnc_id}}" target="_blank">{{gene.hgnc_id}}</a>
```
Comment (northwestwitch) [inline scout/server/blueprints/panels/templates/panels/panel_pdf_simple.html:61]: Link was broken

File: scout/server/blueprints/panels/controllers.py [hunk 4]
```diff
@@ -324,9 +324,9 @@ def downloaded_panel_name(panel_obj, format) -> str:
 
 def panel_data(store, panel_obj):
     """Preprocess a panel of genes."""
+
     panel_obj["institute"] = store.institute(panel_obj["institute"])
-    full_name = "{}({})".format(panel_obj["display_name"], panel_obj["version"])
-    panel_obj["name_and_version"] = full_name
+    panel_obj["name_and_version"] = "{}({})".format(panel_obj["display_name"], panel_obj["version"])
```
Comment (dnil) [inline scout/server/blueprints/panels/controllers.py:329]: ✅

File: scout/server/blueprints/panels/templates/panels/panel_pdf_simple.html [hunk 5]
```diff
@@ -45,27 +48,31 @@ <h4>Scout - Gene panel report</h4> - created on:&nbsp;<strong>{{report_created_a
...
-        <td>Gene name</td>
-        <td>Disease associated transcripts</td>
-        <td>Reduced penetrance</td>
-        <td>Mosaicism</td>
-        <td>Entry version</td>
-        <td>Inheritance</td>
+        <th></th>
+        <th>Gene</th>
+        <th>HGNC ID</th>
+        <th>Disease-associated transcripts</th>
```
Comment (dnil) [inline scout/server/blueprints/panels/templates/panels/panel_pdf_simple.html:54]: 💯

File: scout/server/blueprints/panels/templates/panels/panel_pdf_simple.html [hunk 6]
```diff
@@ -45,27 +48,31 @@ <h4>Scout - Gene panel report</h4> - created on:&nbsp;<strong>{{report_created_a
...
+        <th>Reduced penetrance</th>
+        <th>Mosaicism</th>
+        <th>Entry version</th>
+        <th>Manual inheritance (pre-set terms)</th>
+        <th>Manual inheritance (free text terms)</th>
+        <th>Comment</th>
```
Comment (dnil) [inline scout/server/blueprints/panels/templates/panels/panel_pdf_simple.html:60]: 👍

**Review by dnil — APPROVED** [review]
Looks good!

**Pull request merged**